hudi-cli/src/main/java/org/apache/hudi/cli/commands/HoodieSyncCommand.java

```diff
@@ -60,9 +55,12 @@ public String validateSync(
       @CliOption(key = {"hivePass"}, mandatory = true, unspecifiedDefaultValue = "",
           help = "hive password to connect to") final String hivePass)
       throws Exception {
+    if (HoodieCLI.syncTableMetadata == null) {
+      throw new HoodieException("Sync validate request target table not null.");
+    }
```

Comment:
@nsivabalan Here is replace of `sync validate` check.  `syncTableMetadata` is target table meta client.

Reply:
ok